problems/4.arrays.js

```diff
@@ -2,8 +2,12 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
-
+function firstAndLastAverage(arr){
+  (arr.length>2);{
```

Comment:
if you use pop() and shift() you have to invoke them. Best way to access the first element is arr[0] and best way to access last element is arr[arr.length - 1]